resources/META-INF/plugin.xml

```diff
@@ -21,6 +21,17 @@
 
   <change-notes>
     <![CDATA[
+
+0.12.0:
```

Comment:
I'd rather not personally.  `1.0` suggests something more than we want to (I think).  If you guys feel strongly though I won't resist.